src/chapters/angular-lsn1/studio.rst

```diff
@@ -1,62 +1,197 @@
+2. Type in the text ``Add components here...`` into ``app.component.html``.
+3. View the app in your browser to verify the words "Add components here..." is the only thing that appears on the page.
+
+Header Component
+^^^^^^^^^^^^^^^^
+You need to create a component that shows the title, mission name, and carrier rocket.
+
+1. Create a header component using ``ng g component header``.
+2. In file ``/mission-planning-dashboard/src/app/header/header.component.html`` add HTML:
+
+.. sourcecode:: html+ng2
+   :linenos:
+
+   <h1>Mission Planning Dashboard</h1>
+   <h3>Mission Name: {{ missionName }}</h3>
+   <h3>Carrier Rocket: {{ rocketName }}</h3>
+
+3. Add variables ``missionName`` and ``rocketName`` to the header component in ``mission-planning-dashboard/src/app/header/header.component.ts``
+
+.. sourcecode:: TypeScript
+
+   export class HeaderComponent implements OnInit {
+
+       missionName: string = "Mars 2030";
+       rocketName: string = "Plasma Max";
+
+4. Add a reference to the header component in ``mission-planning-dashboard/src/app/app.component.html``.
+
+.. sourcecode:: html+ng2
+
+   <app-header></app-header>
+
+
+5. View the app in your browser to verify that the title, mission name, and rocket name are visible.
+
+.. figure:: ./figures/example-header-component.png
+   :alt: Screen shot of browser showing address localhost:4200, which has a title of Mission Planning Dashboard, a Mission Name, and a Carrier Rocket.
+
+Crew Component
+^^^^^^^^^^^^^^
+Next you need to make a component to show a list of crew members.
+
+1. Create the component by running ``ng g component crew``.
+2. Set the contents of to be:
+
+.. sourcecode:: html
+
+   <h3>Crew</h3>
+   <ul>
+      <li>Jessica Watkins</li>
+      <li>Raja Chari</li>
+      <li>Jasmin Moghbeli</li>
+   </ul>
 
-Add some CSS to change add different colors, fonts, borders, etc. to your
-dashboard.
+3. Add a reference to the header component in ``mission-planning-dashboard/src/app/app.component.html``.
 
-Add links to information pages about the different countries represented by
-the crew.
+.. sourcecode:: html+ng2
+
+   <app-header></app-header>
+   <app-crew></app-crew>
+
+.. figure:: ./figures/example-crew-component.png
+   :alt: Screen shot of browser showing address localhost:4200, which has a title of Mission Planning Dashboard, a Mission Name,a Carrier Rocket, a Crew header, and a list of crew members in an unordered list.
+
+Equipment Component
+^^^^^^^^^^^^^^^^^^^
+Now you need to create a component to show a list of equipment.
+
+1. Create an equipment component named ``equipment``
+2. In the component display data:
+
+   * An ``<h3>`` that contains "Equipment"
+   * A ``<ul>`` that contains ``<li>`` for: Habitat dome, Drones, Food containers, Oxygen tanks
+
+2. Add the component to ``mission-planning-dashboard/src/app/app.component.html`` using the below HTML. Notice the ``<div>`` surrounding the crew and equipment components.
```

Comment:
This is the second step 2 of this section...

Also, minor rephrasing, "Add the ``equipment`` component to ``app.component.html`` using the HTML below. Notice the ``<div>`` surrounding the crew and equipment components."